cpp/include/cudf/detail/aggregation/aggregation.cuh

```diff
@@ -117,11 +118,12 @@ struct update_target_element<Source,
 };
 
 template <typename Source, bool target_has_nulls, bool source_has_nulls>
-struct update_target_element<Source,
-                             aggregation::MAX,
-                             target_has_nulls,
-                             source_has_nulls,
-                             std::enable_if_t<is_fixed_width<Source>()>> {
+struct update_target_element<
+  Source,
+  aggregation::MAX,
+  target_has_nulls,
+  source_has_nulls,
+  std::enable_if_t<is_fixed_width<Source>() && !is_fixed_point<Source>()>> {
```

Comment:
I'd suggest adding a new `is_primitive_type` trait and use that here instead.

Reply:
Done :)